src/demo/data.js

```diff
@@ -2,51 +2,153 @@ import faker from 'faker/locale/en';
+    ...enrollmentsSample.map(enrollment => ({ ...enrollment, consent_granted: false })), // eslint-disable-line max-len
+    ...enrollmentsSample.map(enrollment => ({ ...enrollment, passed_timestamp: dateInPastMonths, last_activity_date: dateInPastMonths })), // eslint-disable-line max-len
+    ...enrollmentsSample.map(enrollment => ({ ...enrollment, course_end: dateInPastMonth })), // eslint-disable-line max-len
+  ];
+  return enrollmentsData;
+};
+
+const enrollments = getEnrollments();
+
+const filterEnrolledLearners = () => enrollments.filter(enrollment => enrollment.consent_granted === true).map((enrollment) => { // eslint-disable-line max-len
+  const totalCurrentEnrollements = enrollments.filter(currentEnrollment => currentEnrollment.user_email === enrollment.user_email).length; // eslint-disable-line max-len
+  return {
+    user_email: enrollment.user_email,
+    user_account_creation_timestamp: faker.date.past(),
+    enrollment_count: totalCurrentEnrollements,
+  };
+});
+
+const filterEnrolledLearnersForInactiveCourses = () => enrollments.filter(enrollment => enrollment.course_end.getTime() < new Date().getTime()).map((enrollment) => { // eslint-disable-line max-len
+  const totalCurrentEnrollements = enrollments.filter(currentEnrollment => currentEnrollment.user_email === enrollment.user_email).length; // eslint-disable-line max-len
+  const totalCompletedEnrollements = enrollments.filter(currentEnrollment => currentEnrollment.has_passed === true && currentEnrollment.user_email === enrollment.user_email).length; // eslint-disable-line max-len
   return {
-    id: idx,
-    user_email: faker.internet.email(firstName, lastName, provider),
-    course_title: course.title,
-    course_price: course.price,
-    course_start: courseStart,
-    course_end: courseEnd,
-    passed_timestamp: Math.random() <= hasPassed ? faker.date.between(courseStart, courseEnd)
-      : null,
-    has_passed: hasPassed,
-    current_grade: grade / 100,
-    last_activity_date: faker.date.recent(Math.random() <= percentActive ? 7 : 45),
+    user_email: enrollment.user_email,
+    enrollment_count: totalCurrentEnrollements,
+    course_completion_count: totalCompletedEnrollements,
+    last_activity_date: enrollment.last_activity_date,
   };
 });
 
+const filterUnenrolledRegisteredLearners = () => enrollments.map(enrollment => ({
+  user_email: enrollment.user_email,
+  user_account_creation_timestamp: faker.date.past(),
+}));
+
+const filterLearnerActivity = (learnerEnrollments, filterOption) => {
+  let filteredLearnerEnrollments = learnerEnrollments;
+  const lastActivityDate = new Date();
+  if (filterOption === 'active_past_week') {
+    lastActivityDate.setDate(lastActivityDate.getDate() - 7);
+    filteredLearnerEnrollments = learnerEnrollments.filter(enrollment => enrollment.last_activity_date.getTime() >= lastActivityDate.getTime()); // eslint-disable-line max-len
```

Comment:
Can this line (and the others in this function) be formatted to avoid using `// eslint-disable-line max-len`? It would probably also make it easier to read.